lib-injection/sitecustomize.py

```diff
@@ -12,41 +12,86 @@ def _configure_ddtrace():
 
     bootstrap_dir = os.path.abspath(os.path.dirname(ddtrace.bootstrap.sitecustomize.__file__))
     prev_python_path = os.getenv("PYTHONPATH", "")
-    os.environ["PYTHONPATH"] = "%s%s%s" % (bootstrap_dir, os.path.pathsep, prev_python_path)
+    new_python_path = "%s%s%s" % (bootstrap_dir, os.path.pathsep, prev_python_path)
+    os.environ["PYTHONPATH"] = new_python_path
 
     # Also insert the bootstrap dir in the path of the current python process.
     sys.path.insert(0, bootstrap_dir)
-    print("datadog autoinstrumentation: successfully configured python package", file=sys.stderr)
+    print("[info] datadog autoinstrumentation: successfully configured python package, python path is %r" % new_python_path, file=sys.stderr)
 
 
 # Avoid infinite loop when attempting to install ddtrace. This flag is set when
 # the subprocess is launched to perform the installation.
 if "DDTRACE_PYTHON_INSTALL_IN_PROGRESS" not in os.environ:
+
+    # First try to import ddtrace from the current PYTHONPATH
+    # This should always fail as we don't expect the user to have already installed ddtrace.
     try:
         import ddtrace  # noqa: F401
 
-    except ImportError:
-        import subprocess
+    except ModuleNotFoundError:
```

Comment:
ModuleNotFoundError was introduced in [python3.6](https://docs.python.org/3/library/exceptions.html#ModuleNotFoundError). This would raise a syntax error in python2.7 and 3.5

```suggestion
    except ImportError:
```